src/locales/en-US.json

```diff
@@ -1810,7 +1810,7 @@
       "message": "For users with an HMC attached, if you try to disconnect it from this page and the HMC is active, it will attempt to reconnect. The proper way to disconnect a working HMC is to disconnect it from the HMC itself."
     },
     "modal": {
-      "disconnectMessage": "Are you sure you want to disconnect %{count} session? This action cannot be undone. | Are you sure you want to disconnect %{count} sessions? This action cannot be undone.",
+      "disconnectMessage": "Are you sure you want to disconnect 1 session? This action cannot be undone. | Are you sure you want to disconnect %{count} sessions? This action cannot be undone.",
```

Comment:
@vedangimittal, This change is already present in other languages. So, I would prefer us using the same Message and avoid rework for this in translation files.
Eg: https://github.ibm.com/openbmc/webui-vue/blob/1060-ghe/src/locales/de.json